applications/tari_console_wallet/src/main.rs

```diff
@@ -100,7 +100,7 @@ fn main_inner() -> Result<(), ExitError> {
     )?;
 
     #[cfg_attr(feature = "libtor", allow(unused_mut))]
-    let mut config = ApplicationConfig::load_from(&cfg)?;
+    let config = ApplicationConfig::load_from(&cfg)?;
```

Comment:
This mut is used by libtor
Clippy fix here: https://github.com/tari-project/tari/pull/4425

Reply:
Thanks, implemented